views-fieldset/src/main/java/io/micronaut/views/fields/elements/InputPasswordFormElement.java

```diff
@@ -0,0 +1,91 @@
+import io.micronaut.views.fields.InputType;
+import io.micronaut.views.fields.messages.Message;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Input Password.
+ * @see <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password">Input Password</a>
+ * @param name Name of the form control. Submitted with the form as part of a name/value pair
+ * @param id It defines an identifier (ID) which must be unique in the whole document
+ * @param placeholder The placeholder attribute is a string that provides a brief hint to the user as to what kind of information is expected in the field.
+ * @param required If true indicates that the user must specify a value for the input before the owning form can be submitted.
+ * @param readOnly A Boolean attribute which, if present, means this field cannot be edited by the user.
+ * @param maxLength The maximum string length that the user can enter into the text input.
+ * @param minLength The minimum string length that the user can enter into the text input.
+ * @param pattern The pattern attribute, when specified, is a regular expression that the input's value must match for the value to pass constraint validation.
+ * @param size The size attribute is a numeric value indicating how many characters wide the input field should be.
+ * @param value The value attribute of the input element
+ * @param label the input label
+ * @param errors errors associated with this input
+ *
+ * @author Sergio del Amo
+ * @since 4.1.0
+ */
+@Experimental
+@Introspected(builder = @Introspected.IntrospectionBuilder(builderClass = InputPasswordFormElement.Builder.class))
+public record InputPasswordFormElement(@NonNull String name,
+                                       @Nullable String id,
+                                       @Nullable String placeholder,
+                                       boolean required,
+                                       boolean readOnly,
+                                       @Nullable Number maxLength,
+                                       @Nullable Number minLength,
+                                       @Nullable String pattern,
+                                       @Nullable Integer size,
+                                       @Nullable String value,
+                                       @Nullable Message label,
+                                       @NonNull List<Message> errors) implements InputFormElement, FormElementAttributes, GlobalAttributes, InputStringFormElement {
+
+    @Override
+    @NonNull
+    public String getType() {
+        return InputType.ATTR_TYPE_PASSWORD;
+    }
+
+    /**
+     *
+     * @return the Input Password FormElement Builder
+     */
+    @NonNull
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    /**
+     * Input Password Builder.
+     */
+    public static class Builder extends InputStringFormBuilder<InputPasswordFormElement, Builder> {
```

Comment:
can be a `final`

Reply:
resolved via resolved via https://github.com/micronaut-projects/micronaut-views/pull/613/commits/56fc641315f8db4d2471cac1d3dbfebbbe374488